include/pmacc/kernel/atomic.hpp

```diff
@@ -163,9 +163,9 @@ namespace pmacc
              * The id provider for particles is the only code where atomicAllInc is used without an accelerator.
              * @todo remove the unsafe faked accelerator
```

Comment:
please remove this comment. 
I think this todo is now fixed.

Reply:
No the fake acc to be able to execute alpaka atomic functions is still in use but the way how this object has changed so that the compiler is not complaining about a potential uninitilized object.